apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SoftwareCheckStep.tsx

```diff
@@ -21,66 +19,114 @@ export type Props = {
   onComplete: () => void;
 };
 
+// The commands needs to be defined outside of the function component, to avoid creating it at
+// each render, and re-triggering a run for their associated hooks
+const getGenuineCheckFromDeviceIdCommand = command("getGenuineCheckFromDeviceId");
+const getLatestAvailableFirmwareFromDeviceIdCommand = command(
+  "getLatestAvailableFirmwareFromDeviceId",
+);
+
 const SoftwareCheckStep = ({ isDisplayed, onComplete }: Props) => {
   const [genuineCheckStatus, setGenuineCheckStatus] = useState<SoftwareCheckStatus>(
     SoftwareCheckStatus.inactive,
   );
   const [firmwareUpdateStatus, setFirmwareUpdateStatus] = useState<SoftwareCheckStatus>(
     SoftwareCheckStatus.inactive,
   );
+  const [availableFirmwareVersion, setAvailableFirmwareVersion] = useState<string>("");
+
+  const device = useSelector(getCurrentDevice);
+  console.log(device);
```

Comment:
If you don't need those console log for the error state remove them, else put `// TODO: remove` above it so we don't forget